nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -477,19 +477,24 @@ public Description matchSwitch(SwitchTree tree, VisitorState state) {
       return Description.NO_MATCH;
     }
 
-    ExpressionTree switchExpression = tree.getExpression();
-    if (switchExpression instanceof ParenthesizedTree) {
-      switchExpression = ((ParenthesizedTree) switchExpression).getExpression();
+    ExpressionTree switchSelectorExpression = tree.getExpression();
```

Comment:
This is just a variable rename (to avoid confusion with switch expressions) and also a comment to explain why parens are being removed